installation/language/en-GB/joomla.ini

```
@@ -90,6 +90,7 @@ INSTL_DATABASE_NAME_MSG_POSTGRES="The database name is invalid. It must start wi
 INSTL_DATABASE_NO_SCHEMA="No database schema exists for this database type."
 INSTL_DATABASE_PASSWORD_DESC="Either a password you created or a password provided by your host."
 INSTL_DATABASE_PREFIX_DESC="Enter a table prefix or use the randomly generated one."
+INSTL_DATABASE_PREFIX_DUPLICATE_DESC="If you are using an existing database with tables with the same prefix, Joomla will rename those existing tables by adding the prefix "bak_"."
```

```suggestion
INSTL_DATABASE_PREFIX_DUPLICATE_DESC="If you are using an existing database with tables with the same prefix, Joomla will rename those existing tables by adding the prefix \"bak_\"."
```
Shouldn't double quotes inside the string be escaped?

---

I'm using single quotes in my extension to be on the safe side.

---

Whatever ... but we can't leave it as it is now.